sphinx/config.py

```diff
@@ -51,6 +51,8 @@ class Config(object):
         # the real default is locale-dependent
         today_fmt = (None, 'env', [str]),
 
+        figure_language = (None, 'env', [str]),
+        figure_language_filename = ('%(root)s.%(language)s%(ext)s', 'env', [str]),
```

Comment:
This does not need the `[str]` type hint, because its default is of type string as well.

Also, %-based string interpolation looks a bit obtuse here.  Maybe switching for `str.format` makes this a little nicer?

Now that I think about it, how would I implement looking into different directories with this (say, `_locale/fr/images/myimage.png`)?